doc/getting-started.rst

```diff
@@ -405,7 +405,7 @@ run without this option.
 Pinned versions
 _______________
 
-You can also pin required versions in two ways.  You can specify them
+You can also pin required versions, and so so in two ways.  You can specify them
```

Comment:
"and so so"? That doesn't look OK.

Reply:
Indeed, this should be ".., and do so in two ways." !